backends/verilog/verilog_backend.cc

```diff
@@ -1053,6 +1053,16 @@ bool dump_cell_expr(std::ostream &f, std::string indent, RTLIL::Cell *cell)
 		f << stringf(";\n");
 		return true;
 	}
+	
+	if (cell->type == ID($_BUF_)) {
+		f << stringf("%s" "assign ", indent.c_str());
+		dump_sigspec(f, cell->getPort(ID::Y));
+		f << stringf(" = ");
+		dump_attributes(f, "", cell->attributes, " ");
```

Comment:
Will these attributes get attached to anything? For e.g. `$_NOT_` we dump the attributes as a suffix to the `~` operator, so they get attached to that, but here I don't know if they are of any significance.

Reply:
`assign a = (* foo *) b;` turns out to be invalid Verilog. Since it doesn't look like there's any place else appropriate that we can put the attribute, we will simply need to drop it. (`opt_clean` would drop the attribute anyway when it cleans up `$_BUF_` cells.)